internal/resto/client.go

```diff
@@ -164,11 +165,11 @@ func (c *Client) ReadAllowedCCY(ctx context.Context) (int, error) {
 
 // IsTunnelRunning checks whether tunnelID is running. If not, it will wait for the tunnel to become available or
 // timeout. Whichever comes first.
-func (c *Client) IsTunnelRunning(ctx context.Context, id string, wait time.Duration) error {
+func (c *Client) IsTunnelRunning(ctx context.Context, id, parent string, wait time.Duration) error {
```

Comment:
You might as well use the "new" terminology and call it `owner` instead of `parent`.